### PR TITLE
Created arguments for authorizer and description

### PR DESCRIPTION
**Pull request by rmadamson**
Went through the commands and made passing description as well
as the 'authorizer' available via the command line.  Hopefully
this means that in the future, passwords can be dumped into
pkpass via stdin without exposing them via command line argu-
ments.  Everything required to create a password except for
the password itself is now avaialble to input on the command
line.

### CODE REVIEW

File: libpkpass/commands/create.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """This module allows for the creation of passwords"""
-import getpass
+import getpass, sys
```
Comment (ginsburgnm) [inline libpkpass/commands/create.py:2]: We prefer imports on separate lines https://legacy.python.org/dev/peps/pep-0008/#imports

File: libpkpass/commands/create.py [hunk 2]
```diff
@@ -11,23 +11,29 @@ class Create(Command):
...
         """ Run function for class.                                      """
         ####################################################################
-        password1 = getpass.getpass("Enter password to create: ")
-        if password1.strip() == "":
-            raise BlankPasswordError
-        password2 = getpass.getpass("Enter password to create again: ")
-        if password1 != password2:
-            raise PasswordMismatchError
+        if not self.args['description']:
+            self.args['description'] = input("Description: ")
```
Comment (ginsburgnm) [inline libpkpass/commands/create.py:21]: I like this `description` `authorizer` addition, users should be able to throw this into their rc file and get prompted less.

File: libpkpass/commands/generate.py [hunk 3]
```diff
@@ -13,19 +13,24 @@ class Generate(Command):
...
     def _run_command_execution(self):
         """ Run function for class.                                      """
         ####################################################################
         safe, owner = self.safety_check()
+        if not self.args['description']:
+            self.args['description'] = input("Description: ")
+
+        if not self.args['authorizer']:
+            self.args['authorizer'] = input("Authorizer: ")
+
```
Comment (ginsburgnm) [inline libpkpass/commands/generate.py:29]: Is there a reason for placing this outside of the if check? This means a user might get prompted for additional information even if the `NotThePasswordOwnerError` is going to fire.

File: libpkpass/commands/create.py [hunk 4]
```diff
@@ -11,23 +11,29 @@ class Create(Command):
...
 
-        description = input("Description: ")
-        authorizer = input("Authorizer: ")
-        self.create_or_update_pass(password1, description, authorizer)
+        if not self.args['authorizer']:
+            self.args['authorizer'] = input("Authorizer: ")
 
+        if not self.args['stdin']:
```
Comment (ginsburgnm) [inline libpkpass/commands/create.py:26]: Can we reorder this, such that we don't change the user's flow?

Comment (rmadamson) [reply libpkpass/commands/create.py:26]: Yes

File: libpkpass/commands/generate.py (hunk 3 above)
Comment (rmadamson) [reply libpkpass/commands/generate.py:29]: Sorry, I see what you mean.  Added back in the right place

File: libpkpass/commands/create.py (hunk 1 above)
Comment (rmadamson) [reply libpkpass/commands/create.py:2]: Sounds good

**Comment by rmadamson** [comment]
I think I've addressed all of your comments @ginsburgnm

**Pull request merged**